plugins/modules/system/sudoers.py

```diff
@@ -145,7 +149,12 @@ def exists(self):
 
     def matches(self):
         with open(self.file, 'r') as f:
-            return f.read() == self.content()
+            content_matches = f.read() == self.content()
+
+        current_mode = oct(os.stat(self.file).st_mode & 0o777)
+        mode_matches = current_mode == oct(self.FILE_MODE)
```

Comment:
Out of curiosity: why apply `oct()` to both sides, but kind of obfuscated in two lines? Why not simply:
```suggestion
        current_mode = os.stat(self.file).st_mode & 0o777
        mode_matches = current_mode == self.FILE_MODE
```
?